### ISSUE TITLE
Unable to deploy to supabase edge fucntions

### ISSUE THREAD

**Issue by ak4zh** [lawrencecchen/stripe-sync#1]
Anyway, we can reduce the size of the library?
It seems supabase has a 2MB limit on edge functions.

```bash
Error: Failed to create a new Function on the Supabase project: {"message":"The function size exceeds the allowable limit"}```

**Comment by lawrencecchen** [comment]
Thanks for bringing this up! What version of Stripe are you on?

**Comment by ak4zh** [comment]
```js
import { serve } from "https://deno.land/std@0.131.0/http/server.ts";
import Stripe from "https://esm.sh/stripe@9.6.0?target=deno&no-check";
import {
  createDenoHandler,
  createSupabaseAdapter,
} from "https://esm.sh/stripe-sync@0.0.9";
import { createClient } from "https://esm.sh/@supabase/supabase-js@2.0.0-rc.4";
```

This imports results in following error:
```bash
Error: Failed to create a new Function on the Supabase project: {"message":"The function size exceeds the allowable limit"}
```

With the following imports I get typescript issues:

```js
import { serve } from "https://deno.land/std@0.131.0/http/server.ts";
import Stripe from "https://esm.sh/stripe@9.6.0?target=deno&no-check";
import {
  createDenoHandler,
  createSupabaseAdapter,
} from "https://esm.sh/stripe-sync@0.0.11";
import { createClient } from "https://esm.sh/@supabase/supabase-js@2.1.0";

```

```bash
Error: Error bundling function: exit status 1
error: TS2322 [ERROR]: Type 'SupabaseClient<any, "stripe", any>' is not assignable to type 'SupabaseClient<any, string, any>'.
  Property 'supabaseUrl' is protected but type 'SupabaseClient<Database, SchemaName, Schema>' is not a class derived from 'SupabaseClient<Database, SchemaName, Schema>'.
                supabase: supabaseClient
                ~~~~~~~~
    at file:///home/ak4zh/CodeProjects/sveltekit-saas-starter/supabase/functions/stripe-sync/index.ts:27:3

    The expected type comes from property 'supabase' which is declared here on type '{ supabase: SupabaseClient<any, string, any>; schema?: string | undefined; }'
        supabase: SupabaseClient<any, string, any>;
        ~~~~~~~~
        at https://esm.sh/v98/stripe-sync@0.0.11/dist/index.d.ts:9:5
```

**Comment by lawrencecchen** [comment]
Should be fixed on 0.1.3. Update imports to:

```ts
import { serve } from "https://deno.land/std@0.131.0/http/server.ts";
import { createClient } from "https://esm.sh/@supabase/supabase-js@2.1.0";
import { createDenoHandler } from "https://esm.sh/stripe-sync@0.1.3/adapters/deno";
import { createSupabaseAdapter } from "https://esm.sh/stripe-sync@0.1.3/databaseAdapters/supabase";
import Stripe from "https://esm.sh/stripe@10.17.0?target=deno&no-check";
import invariant from "https://esm.sh/tiny-invariant@1.3.1";
```

TypeScript seems to be broken on Deno, but it works in Node. Feel free to contribute if you can!
